build.sbt

```diff
@@ -9,6 +9,7 @@ libraryDependencies ++= Seq(
   "org.jetbrains.dokka" % "dokka-core" % dokkaVersion,
   "org.jetbrains.dokka" % "dokka-test-api" % dokkaVersion,
   "org.jetbrains.kotlinx" % "kotlinx-html-jvm" % kotlinxVersion,
+  "com.virtuslab.dokka" % "dokka-site" % "0.1.6",
```

Comment:
Maybe we should extract version to `val`? It would be more transparent and users may want to change this version to published locally